ChangeLog.md

```diff
@@ -34,7 +34,7 @@
 ### Added
 
 - Adds the `cardano-address key walletid` command, which derives a
-  [`cardano-wallet` wallet ID](https://IntersectMBO.github.io/cardano-wallet/design/WalletId)
+  [`cardano-wallet` wallet ID](https://cardano-foundation.github.io/cardano-wallet/design/WalletId)
```

Comment:
Hm... still doesn't seem to work but not sure where this should point to 🤷‍♂️

Reply:
probably should be this https://input-output-hk.github.io/adrestia/cardano-wallet/design/WalletId

Reply:
changed